bazel/external/lightstep.sh

```diff
@@ -0,0 +1,20 @@
+# This is a Bazel genrule script. It acts like Bash, but has some extra
```

Comment:
Maybe have the extensions of these `.genrule.cmd` or something, so that it can't be confused with real `.sh`.

Reply:
Done.